script/lazytox.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python
```

Comment:
Though I hate it when people still use python 2, there are quite a few around (including me for certain outdated scientific libraries 😭...). For a subset of those people `which python` will point to a python 2 installation. I guess it's better to do this instead:

```python
#!/usr/bin/env python3
```